src/main/java/com/flowable/flowableboot/service/UmsTaskService.java

```diff
@@ -0,0 +1,68 @@
+package com.flowable.flowableboot.service;
+
+import com.flowable.flowableboot.model.UmsTask;
+import com.flowable.flowableboot.repository.UmsTaskRepository;
+import org.flowable.engine.RuntimeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+
+@Service
+public class UmsTaskService {
+    private UmsTaskRepository umsTaskRepository;
+    private RuntimeService runtimeService;
+
+    @Autowired
+    public UmsTaskService(RuntimeService runtimeService, UmsTaskRepository umsTaskRepository){
+        this.runtimeService = runtimeService;
+        this.umsTaskRepository = umsTaskRepository;
+    }
+
+    /**
+     * Create a new UmsTask - Ideally, this would assume a process was started in workflow engine
+     *
+     * @param process_instance_id process_instance_id
+     * @param name name
+     * @param requester requester
+     * @param assignee assignee
+     * @param priority priority
+     * @param dueDate dueDate
+     * @param receivedDate receivedDate
+     * @param loe loe
+     * @param status status
+     * @param description description
+     *
+     * @return UmsTask Entity
+     */
+    public UmsTask createUmsTask(String process_instance_id, String name, String requester, String assignee, Integer priority,
```

Comment:
This should return void specifically, and the controller should return just 200 code on successful task creation.  Returning objects when saved can create anti-patterns in the front end by encouraging them to keep a running, stateful list of objects.  The flow should look more like:

1) Fetch a list of relevant umsTasks by requestor
2) Save a new umsTasks - receive either a 200 OK, or the relevant error code. 
3) Fetch a list of relevant umsTasks by requestor again, and receive the list of which the new task is a member.

Reply:
This makes sense to me....the return was updated to void